packages/create-sitecore-jss/src/templates/nextjs-sxa/src/lib/middleware/plugins/redirects.ts

```diff
@@ -13,6 +15,7 @@ class RedirectsPlugin implements MiddlewarePlugin {
       apiKey: config.sitecoreApiKey,
       siteName: config.jssAppName,
     });
+    this.locales = nextConfig().i18n.locales;
```

Comment:
`locales` should be owned by `RedirectsMiddleware` (another option on `RedirectsMiddlewareConfig`)